cli/pcluster/config/mappings.py

```diff
@@ -634,6 +698,12 @@
                 "type": EBSSettingsParam,
                 "referred_section": EBS,
                 "validators": [ebs_settings_validator],
+                "update_policy": UpdatePolicy(
+                    UpdatePolicy.UNSUPPORTED,
+                    fail_reason="Cannot add/remove EBS Sections",
+                    action_needed="Make sure you are using the same number of EBS sections and restore all shared_dir "
+                                  "parameters to old values"
```

Comment:
Due to the introduction of sections labels persistence, this message is obsolete. It could be now the same default message for UNSUPPORTED changes:
""Restore {0} value to '{1}'. If you need this change, please consider creating "
    "a new cluster instead of updating the existing one."
Do you think it's OK?